etc/singularity.conf.in

```diff
@@ -195,3 +195,11 @@ bind path = /etc/hosts
 # errors when accessed from container (typically bind mounts)
 #autofs bug path = /nfs
 #autofs bug path = /cifs-share
+
+
+# MEMORY FS TYPE: [tmpfs/ramfs]
+# DEFAULT: tmpfs
+# This feature allow to choose temporary filesystem type used by Singularity.
+# Cray CLE has issue (kernel panic) when Singularity use tmpfs, so on Cray CLE
+# it's recommended to set this value to ramfs to avoid kernel panic
+@MEMORY_FS_TYPE@ = @MEMORY_FS_TYPE_DEFAULT@
```

Comment:
This... doesn't work does it? These should be replaced by autoconf, not from the `config_default.h` defines.

So, wouldn't you need something like the following in `configure.ac`:
```
AC_DEFINE(MEMORY_FS_TYPE,"memory fs type")
AC_DEFINE(MEMORY_FS_TYPE_DEFAULT,"tmpfs")
```

And then change `config_defaults.h.in` to be something like:
```
#define MEMORY_FS_TYPE "@MEMORY_FS_TYPE@"
#define MEMORY_FS_TYPE_DEFAULT "@MEMORY_FS_TYPE_DEFAULT@"
```

Reply:
Yes it works, singularity.conf is generated by configure_transform.py in https://github.com/singularityware/singularity/blob/d6e815477f1ce75389ea79dfc590390513c7102d/etc/Makefile.am#L7